data-service/src/test/java/com/amazonaws/blox/dataservice/integration/DataServiceIntegrationTest.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.dataservice.integration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.amazonaws.blox.dataservicemodel.v1.model.EnvironmentId;
+import com.amazonaws.blox.dataservicemodel.v1.model.wrappers.*;
+import java.util.Arrays;
+import org.junit.Test;
+
+public class DataServiceIntegrationTest extends DataServiceIntegrationTestBase {
```

Comment:
Not sure `DataServiceIntegrationTest` is a good name,  I would suggest we move separate each API into its own test class. Even they are just happy path, but we potentially could add more tests, such as exception handling cases.  Thoughts?

Reply:
That makes sense. I can break it down into separate tests for each API, so later people can add the non-happy case tests in, if necessary.